js/components/skipControls.js

```diff
@@ -274,15 +302,21 @@ class SkipControls extends React.Component {
       return null;
     }
 
-    const className = classNames('oo-skip-controls', {
+    const className = classNames('oo-skip-controls', this.props.className, {
       'oo-inactive': this.props.isInactive,
       'oo-in-background': this.props.isInBackground
     });
     const buttonTemplate = this.getButtonTemplate();
 
+    const skipControlsStyle = {};
+    if (this.props.maxWidth) {
```

Comment:
Is it necessary to set this using an inline style? Could this style be added to the custom class?

Reply:
Added